dockers/docker-sonic-p4rt/Dockerfile.j2

```diff
@@ -0,0 +1,31 @@
+{% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
+FROM docker-config-engine-buster
+
+ARG docker_container_name
+ARG git_commit
+RUN [ -f /etc/rsyslog.conf ] && sed -ri "s/%syslogtag%/$docker_container_name#%syslogtag%/;" /etc/rsyslog.conf
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+{% if docker_sonic_p4rt_debs.strip() -%}
+# Copy locally-built Debian package dependencies
+{{ copy_files("debs/", docker_sonic_p4rt_debs.split(' '), "/debs/") }}
+
+# Install locally-built Debian packages and implicitly install their dependencies
+{{ install_debian_packages(docker_sonic_p4rt_debs.split(' ')) }}
+{%- endif %}
+
+RUN apt-get clean -y      && \
+    apt-get autoclean -   && \
```

Comment:
>- [](http://example.com/codeflow?start=22&length=1)

Is it a mistake, I don't see an short option after dash.

Reply:
Good catch
```suggestion
    apt-get autoclean -y   && \
```

Reply:
Pushed the change.